src/FakeItEasy/Core/DelegateRaiser.cs

```diff
@@ -68,7 +68,7 @@ private static void AssertThatValuesSatisfyCallSignature(MethodInfo callMethod,
 
             throw new FakeConfigurationException(
                 "The event has the signature ({0}), but the provided arguments have types ({1})."
-                    .FormatInvariant(fakeSignature, actionSignature, "Raise"));
+                    .FormatInvariant(fakeSignature, actionSignature, nameof(Raise)));

```

Comment:
Actually, I think `"Raise"` could be removed... there are only 2 placeholders in the format strings, not 3